jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/Response.java

```diff
@@ -1402,7 +1402,7 @@ else if (contentLength > NO_CONTENT_LENGTH)
 
     public static void putHeaders(HttpServletResponse response, HttpContent content, long contentLength, boolean etag)
     {
-        long lml = content.getLastModified().getLongValue();
```

Comment:
The bug seems to be in `HttpContent.getLastModified()`, why not fix it there?

Reply:
The bug isn't in `HttpContent.getLastModified()`, which is a HttpField representing a stringified Date value. If you look at the same code in jetty-10/11 etc, this is how we handle this case. See https://github.com/eclipse/jetty.project/blob/jetty-10.0.x/jetty-server/src/main/java/org/eclipse/jetty/server/Response.java#L1450